RenderStateNotation/src/RenderStateNotationParserImpl.cpp

```diff
@@ -26,12 +26,11 @@
 
 #include "pch.h"
 
-#include "RenderStateNotationParserImpl.hpp"
```

Comment:
The include the header before any other headers, even before std ones is to make sure it compiles OK as a standalone and has all dependent headers in it.